automation_use_cases/scan_automation/config/settings.yml

```diff
@@ -0,0 +1,7 @@
+connection:
+  region: us
+  api_key: 
+scan_info:
```

Comment:
Thoughts on allowing this to be a list of app/scan config pairings so people can define multiple? @jherlitz-r7 @zyoutz-r7

Reply:
I think a list would make sense while the `status_check_interval` global to all of them.